Readme still up-to-date?

When I try to run your example I get some errors in the console.

## steps

```bash
git clone ...
cd ...
sudo make run
```

open browser at http://localhost:1776/#10/39.8087/-86.0879 but only an empty map appears and in the console I read this:

```
Access to fetch at 'https://zelonewolf.github.io/maplibre-shield-rotation-sample/data/v3.json' from origin 'http://localhost:1776' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
zelonewolf.github.io/maplibre-shield-rotation-sample/data/v3.json:1 Failed to load resource: net::ERR_FAILED
evented.js:140 Error: Failed to fetch
    at ajax.js:161
Ct.fire @ evented.js:140
:1776/#10/39.8087/-86.0879:1 Access to fetch at 'https://zelonewolf.github.io/maplibre-shield-rotation-sample/sprites/sprite@2x.json' from origin 'http://localhost:1776' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
zelonewolf.github.io/maplibre-shield-rotation-sample/sprites/sprite@2x.json:1 Failed to load resource: net::ERR_FAILED
evented.js:140 Error: Failed to fetch
    at ajax.js:161
Ct.fire @ evented.js:140
:1776/#10/39.8087/-86.0879:1 Access to fetch at 'https://zelonewolf.github.io/maplibre-shield-rotation-sample/sprites/sprite@2x.png' from origin 'http://localhost:1776' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
zelonewolf.github.io/maplibre-shield-rotation-sample/sprites/sprite@2x.png:1 Failed to load resource: net::ERR_FAILED
```

Is the readme still up-to-date?

--- a/index.html
+++ b/index.html
@@ -54,11 +54,11 @@
         center: [0, 0],
         glyphs: "https://fonts.openmaptiles.org/{fontstack}/{range}.pbf",
         layers: americanaLayers,
-        sprite: "https://zelonewolf.github.io/maplibre-shield-rotation-sample/sprites/sprite",
+        sprite: "http://localhost:1776/sprites/sprite",
         bearing: 0,
         sources: {
           openmaptiles: {
-            url: "https://zelonewolf.github.io/maplibre-shield-rotation-sample/data/v3.json",
+            url: "http://localhost:8080/data/v3.json",
             type: "vector",
           },
           maptiler_attribution: {

